Azure/sonic-buildimage
# [devices]: Fix arista-convertfs for aligning flash via sfdisk in Jessie

The sfdisk in Jessie use unit Cylinder by default. To perform 1M
aligning partition, 2048 sector unit is desirable instead.



**- What I did**
Use "-u S --force" option to perform flash repartitioning so that 1M aligning is achieved with start=2048 sectors. "--force" is needed otherwise old sfdisk in Jessie will complain the aligning
scheme.

**- How I did it**

**- How to verify it**
Install the image on arista 7050 then reboot the box to verify the repartitioning of flash is fine. The first partition in the table should start with 2048 sectors.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
